oak_abi/proto/label.proto

```diff
@@ -82,3 +83,11 @@ message TlsEndpointTag {
   // using the set of Certificate Authorities (CA) that are available to it.
   string authority = 1;
 }
+
+// Policies related to user identification.
+message UserIdentityTag {
```

Comment:
As I am rewriting the chat app to use labels, it turns out that we use public keys to represent things other than users (e.g. chat rooms), so I think we should use a more generic terminology, so perhaps just use `PublicKeyIdentityTag` or similar?

Reply:
Thanks. I'll change the name.

Reply:
I have changed the name in my new PR,  but I now wonder if `PublicKeyIdentityTag` still corresponds to the `user` sub-lattice.  I guess there should be a one-to-one correspondence between the tag types and the principals. Or do we consider `chat room` to have the same principal nature as users?